src/scripts.js

```diff
@@ -347,23 +310,12 @@ function getInstructionsElement(e, selectedRecipe) {
 
 function show(elements) {
   elements.forEach(element => {
-    element.classList.remove('hidden')
-  });function filterRecipesByTag() {
```

Comment:
This was really wonky. We think it must've been a weird copy/paste scenario. Remedied.